src/io/flutter/inspections/WrongModuleTypeNotificationProvider.java

```diff
@@ -33,27 +33,52 @@
 import io.flutter.sdk.FlutterSdkUtil;
 import org.jetbrains.annotations.NotNull;
 
+import java.io.IOException;
 import java.util.Set;
+import java.util.regex.Pattern;
 
 
 public class WrongModuleTypeNotificationProvider extends EditorNotifications.Provider<EditorNotificationPanel> implements DumbAware {
   private static final Key<EditorNotificationPanel> KEY = Key.create("Wrong module type");
   private static final String DONT_ASK_TO_CHANGE_MODULE_TYPE_KEY = "do.not.ask.to.change.module.type"; //NON-NLS
   private static final String FLUTTER_YAML_FILE = "flutter.yaml"; //NON-NLS
+  private static final String PUBSPEC_YAML_FILE = "pubspec.yaml"; //NON-NLS
+
+  private static final Pattern FLUTTER_SDK_DEP = Pattern.compile(".*dependencies:\\s*flutter:\\s*sdk:\\s*flutter"); //NON-NLS
```

Comment:
I was thinking about this on my way in actually.  I'm tempted to keep it super simple and just look for `\\s*sdk:\\s*flutter` since that's the salient bit and I can't imagine many false positives.